### ISSUE TITLE
tsh version ends with error in v9.0.2

### ISSUE THREAD

**Issue by Joerger** [gravitational/teleport#11513]
```
Teleport v9.0.2 git:v9.0.2-0-g354b8c037 go1.17.8
ERROR: open /home/bjoerger/.tsh/config/config.yaml: no such file or directory
```

This will cause issues for any pipelines which use `tsh version`, such as the teleport-plugins integration tests.

**Comment by Valien** [comment]
In the meantime running a `tsh config` solves the problem (at least on OSX)

**Comment by zmb3** [comment]
Closed via #11495

**Comment by Valien** [comment]
FYI @zmb3 : This is broken in `v8.3.5` and `v8.3.6` clients as well. I had to go to `v8.3.4` to get it to not show. See image for details:

![Screen Shot 2022-04-05 at 9 19 26 AM](https://user-images.githubusercontent.com/1424442/161762982-792eee1a-e797-49d1-89fb-38a8d19a5f68.png)

![Screen Shot 2022-04-05 at 9 18 55 AM](https://user-images.githubusercontent.com/1424442/161763008-5ed14508-c0a4-4b99-95b3-2c1c04dcca99.png)

![Screen Shot 2022-04-05 at 9 18 43 AM](https://user-images.githubusercontent.com/1424442/161763028-90ace5e1-3ec6-43ce-9d40-c95e24ccb59b.png)
